src/main/java/lukemccon/airdrop/controllers/PackageController.java

```diff
@@ -28,4 +40,64 @@ public static boolean onCommand(CommandSender sender, String[] args) {
 		return true;
 	}
 
+	public static boolean deletePackageCommand(CommandSender sender, String[] args) {
+		if (args.length != 3 ) {
+			ChatHandler.sendErrorMessage(sender, "Need to specify a package name to delete");
+		}
+
+		String packageName = args[2];
+		try {
+			PackageManager.deletePackage(packageName);
+			ChatHandler.sendMessage(sender, ChatColor.AQUA + packageName + ChatColor.BLUE + " was successfully deleted");
+		} catch (PackageNotFoundException e) {
+			ChatHandler.sendErrorMessage(sender, "Unable to delete package " + ChatColor.DARK_RED + packageName + ChatColor.RED + " not found");
+		}
+		return true;
+	}
+
+
+
+	public static boolean createPackageCommand(CommandSender sender, String[] args) {
+
+			if (args.length != 4) {
+				ChatHandler.sendErrorMessage(sender, "Package create command requires 4 total arguments");
+				ChatHandler.sendErrorMessage(sender, "Example: /airdrop package create myPackage 12.0");
+			}
+
+			// Create a new package
+			if (!(sender instanceof Player)) {
+				ChatHandler.sendErrorMessage(sender,"Must be a player to use this command");
+				return true;
+			}
+
+			Player player = (Player) sender;
+
+			String packageName = args[2];
+			String priceString = args[3];
+			Double price = null;
+
+			if ( packageName == null || packageName.isEmpty()) {
```

Comment:
Should probably be ```isBlank()``` just in case someone somehow manages to try to put whitespace characters as the name, not sure how minecraft deals with multiple spaces mid command, it might register one as the arg.

Reply:
yeah I think that would completely screw up the command, it would throw an error as it would read it as the next arg and that should be a double.
They would get an error that the price is not a double